Poke/App.tsx

```diff
@@ -83,10 +88,14 @@ const App = (props: any) => {
       <SafeAreaView>
         <ScrollView contentInsetAdjustmentBehavior="automatic" style={styles.scrollView}>
           <View style={styles.body}>
-            <Text style={styles.simple}>Bem vindo(a) à Taqtile!</Text>
-            <EmailInput text={email} onTextChange={setEmail} />
-            <PasswordInput text={password} onTextChange={setPassword} />
-            <SubmitButton onTap={() => {handleSubmit(props)}} />
+            {isLoading ? <ActivityIndicator size={'large'} color="#000000"/> : 
+            <> 
+              <Text style={styles.simple}>Bem vindo(a) à Taqtile!</Text>
+              <EmailInput text={email} onTextChange={setEmail} />
+              <PasswordInput text={password} onTextChange={setPassword} />
+              <SubmitButton onTap={() => {handleSubmit(props)}} />
```

Comment:
Por conta da não necessidade de repassar as props:
```suggestion
              <SubmitButton onTap={handleSubmit} />
```